api/tests/integration/infrastructure/repositories/organization-invited-user-repository_test.js

```diff
@@ -0,0 +1,473 @@
+const { expect, databaseBuilder, catchErr, knex, sinon } = require('../../../test-helper');
+const organizationInvitedUserRepository = require('../../../../lib/infrastructure/repositories/organization-invited-user-repository');
+const { NotFoundError } = require('../../../../lib/domain/errors');
+const OrganizationInvitation = require('../../../../lib/domain/models/OrganizationInvitation');
+
+describe('Integration | Repository | OrganizationInvitedUserRepository', function () {
+  afterEach(async function () {
+    await knex('user-orga-settings').delete();
+    await knex('memberships').delete();
+  });
+
+  describe('#get', function () {
+    it('should return an OrganizationInvitedUser userId', async function () {
+      // given
+      const ExpectedUserId = 123;
+      const organizationInvitationId = databaseBuilder.factory.buildOrganizationInvitation().id;
+      databaseBuilder.factory.buildUser({
+        email: 'wrongUser@example.net',
+      });
+      const user = databaseBuilder.factory.buildUser({
+        id: ExpectedUserId,
+        email: 'user@example.net',
+      });
+
+      await databaseBuilder.commit();
+
+      // when
+      const OrganizationInvitedUser = await organizationInvitedUserRepository.get({
+        organizationInvitationId,
+        email: user.email,
+      });
+
+      // then
+      expect(OrganizationInvitedUser.userId).to.equal(ExpectedUserId);
+    });
+
+    it('should return an OrganizationInvitedUser.invitation', async function () {
```

Comment:
```suggestion
it('should return the invitation of the invited user', async function () {
```
J'ai l'impression que les intitulés des `it` sont très techniques ... est-ce qu'on pourrait les orienter de manière un peu plus fonctionnelle ?